public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -173,7 +173,7 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
   private _prevScrollPos?: number;
 
   // TODO: use feature toggle to allow v2 serializer
-  private _serializer: DashboardSceneSerializerLike<Dashboard | DashboardV2Spec> = getDashboardSceneSerializer();
+  public serializer: DashboardSceneSerializerLike<Dashboard | DashboardV2Spec> = getDashboardSceneSerializer();
```

Comment:
Why do we need to make it public? I think serializer should never be accessed directly. If you are doing it for getting the `serializer.getSnapshotUrl` outside of the class, there's alread a public `getSnapshotUrl` on the `DashboardScene` interface.